src/components/ConnectWallet/Wallets/LilyWallet.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <connection-details name="Lily Wallet" requires="bitcoind">
+    <step-list>
+      <step>
+        Make sure Tor is up and running on your system.
+        <b-link v-b-modal.tor-modal>Click here</b-link> for instructions.
+      </step>
+      <step>
+        Open Lily Wallet, click the green circle in the top right corner, and
+        then
+        <span class="font-weight-bold">"Network settings"</span>.
+      </step>
+      <step>
+        Click the and click
+        <span class="font-weight-bold">"Change Data Source"</span> option and
+        then <span class="font-weight-bold">"Connect to Custom Node"</span> to
+        bring up the connection modal.
+      </step>
+      <step>
+        In the <span class="font-weight-bold">"Host"</span>, enter
+        <input-copy
+          class="my-1"
+          :value="`http://${urls.bitcoin.rpc.address}`"
```

Comment:
```suggestion
          :value="`http://${urls.bitcoin.rpc.address}:${urls.bitcoin.rpc.port}`"
```

Reply:
I was thinking the same thing when I was making the PR, but it looks like `urls.bitcoin.rpc.address` returns the appended port as well. Screenshot:

![Screen Shot 2021-04-19 at 9 36 18 AM](https://user-images.githubusercontent.com/1429113/115263489-b4c0dc00-a0f2-11eb-894d-d53cdf3be7a2.png)

Reply:
It does not. See:

https://github.com/getumbrel/umbrel-dashboard/blob/fc3975ecc9f16c8b315be6da9419703ef7d6d259/src/store/modules/bitcoin.js#L21-L27

https://github.com/getumbrel/umbrel-dashboard/blob/fc3975ecc9f16c8b315be6da9419703ef7d6d259/src/store/modules/bitcoin.js#L150-156

And:
https://github.com/getumbrel/umbrel-manager/blob/1635a3a1170c8545ee705dbd2437b1442a83f44a/logic/system.js#L59-L81

Reply:
Sure, sorry about that. You can manually update SSHing into your Pi and running:

```
sudo ~/umbrel/scripts/update/update --repo getumbrel/umbrel#v0.3.8
```